src/endpoints/auth/auth.controller.ts

```diff
@@ -0,0 +1,95 @@
+import { FastifyPluginAsync, FastifyPluginOptions } from 'fastify';
+import { UserEntity, VerificationCodes } from '../../shared/database';
+import Ajv from 'ajv';
+const ajv = new Ajv();
+import jwt from 'jsonwebtoken';
+import { createHmac, randomUUID } from 'crypto';
+import { createTransport } from 'nodemailer';
+
+const secret = process.env.CRYPT_SECRET!;
```

Comment:
What's this? How can you be sure this variable exists? Move to the JWT config service